app/main.py

```diff
@@ -1,13 +1,32 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
+    return {
+        "line": error.get("line_number"),
+        "column": error.get("column_number"),
+        "message": error.get("text"),
+        "name": error.get("code"),
+        "source": "flake8"
+    }
     pass
 
-
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
+    return {"errors":
+        [{format_linter_error(error)} for error in errors],
+    "path": file_path,
+    "status": "passed"
+    if errors == []
+    else "failed"
+    }
     pass
 
-
 def format_linter_report(linter_report: dict) -> list:
-    # write your code here
+    return [
+            {
+                "errors": [
+                    {format_single_linter_file(list(linter_report)[1], errors)
+                    for paths, errors in linter_report.items()}],
+                "path": list(linter_report)[1],
+                "status": "passed"
+                if any("errors")
```

Comment:
The condition 'if any("errors")' is incorrect. 'any' should be used with a list or iterable. You might want to check if 'errors' is empty or not.